mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/plugins/Webjars.kt

```diff
@@ -2,9 +2,12 @@ package no.nav.mulighetsrommet.api.plugins
 
 import io.ktor.server.application.*
 import io.ktor.server.webjars.*
+import no.nav.mulighetsrommet.api.SwaggerConfig
 
-fun Application.configureWebjars() {
-    install(Webjars) {
-        path = "assets"
+fun Application.configureWebjars(swaggerConfig: SwaggerConfig?) {
```

Comment:
Litt jalla løsning for å skru av Swagger i prod, men det var den enkleste løsningen og ta hele Webjars-plugin.